### PR TITLE
Upgrade to canton 0.4.1 and reenable multi-ledger test

### PR DESCRIPTION
**Pull request by oliverse-da**
### Pull Request Checklist

- [X] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [X] Include appropriate tests
- [X] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags, if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-test-tool-on-canton/BUILD.bazel [hunk 1]
```diff
@@ -64,5 +64,5 @@ conformance_test(
         5021,
     ],
     server = ":canton-test-runner-with-dependencies",
-    tags = ["manual"],
+    tags = [],
```
Comment (stefanobaghino-da) [inline ledger/ledger-api-test-tool-on-canton/BUILD.bazel]: If we really want to have this run on CI we can simply remove this line. @SamirTalwar?

Comment (stefanobaghino-da) [reply ledger/ledger-api-test-tool-on-canton/BUILD.bazel]: Ooops, didn't notice he already reviewed. :sweat_smile: Resolving.

Comment (oliverse-da) [reply ledger/ledger-api-test-tool-on-canton/BUILD.bazel]: > If we really want to have this run on CI we can simply remove this line. @SamirTalwar?

👍 - I will try it out and make sure that bazel defaults this properly and does not get upset.

Comment (SamirTalwar) [reply ledger/ledger-api-test-tool-on-canton/BUILD.bazel]: Oh, good point, we can just delete this line.

**Pull request merged**